operator-kustomize/cluster-role.yaml

```diff
@@ -1,76 +0,0 @@
-apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Does the plugin create the cluster role as well? I think it's better to have it explicit, and still think we should keep it